### DISCUSSION TITLE
Using pythonnet netfx on windows arm64

### DISCUSSION THREAD

**Discussion by afaq666** [pythonnet/pythonnet#2523] [Q&A] [answered]
Hi everyone, I am working on a small visual studio c# application that is being compiled for native arm64, it also has some python code.
I installed a python arm64 version and when I install pythonnet and initialize it using core-clr it seems to work fine but when it uses netfx I am getting an error, something like "virtualenv_path_to_clr_loader/amd64/clr_loader.dll" can't be loaded.
I check pip's installation of clrloader and I only see that there are x86 and amd64 versions of the dll, so I guess clr_loader does not support arm64.
I don't want to use .net core, so what should I do to get clr_loader working on arm64? I tried downloading the code for clr_loader and compile as arm64 but when I try to use that dll it fails with a different error that makes me think that clr_loader code is not ready for arm64.

Could you please guide me or give me ideas or suggestions on what could I do to make netfx work on arm64?

Thanks in advance for reading or posting your suggestions.

**Comment by filmor** [comment, marked as answer]
[`clr-loader`](https://github.com/pythonnet/clr-loader) relies on [`NXPorts`](https://github.com/MeikTranel/NXPorts) (for .NET Framework) which in turn uses [`dnlib`](https://github.com/0xd4d/dnlib) to create exported symbols in the assemblies.

`dnlib` doesn't support ARM64 as of now: https://github.com/0xd4d/dnlib/issues/165
